modules/gql/schema.graphql

```diff
@@ -265,6 +265,7 @@ type Query {
   anchorProducer: AnchorProducer
   getCurrentNumber: TestResult # TESTING QUERY
   witnessStake(account: String!): Uint64!
+  findCID(cidString: String!): JSON!
```

Comment:
This should probably be named `getDagByCID` since not all CIDs refer to DAGs. For example, contract code is just stored as raw bytes